storybook/storybook-react/src/Badge/Badge.stories.tsx

```diff
@@ -0,0 +1,30 @@
+/**
+ * @license EUPL-1.2+
+ * Copyright (c) 2024 Gemeente Amsterdam
+ */
+
+import { Badge } from '@amsterdam/design-system-react'
+import { Meta, StoryObj } from '@storybook/react'
+
+const meta = {
+  title: 'Feedback/Badge',
+  component: Badge,
+  args: {
+    label: 'Tip',
+  },
+  argTypes: {
+    color: {
+      control: {
+        type: 'inline-radio',
+      },
+      options: ['blue', 'dark-blue', 'green', 'dark-green', 'orange', 'magenta', 'purple', 'yellow', 'black', 'white'],
+      selected: 'dark-green',
+    },
+  },
+} satisfies Meta<typeof Badge>
+
+export default meta
+
+type Story = StoryObj<typeof meta>
+
+export const Default: Story = {}
```

Comment:
Why delete the example of a Tip in a list? It’s the only extended example we had, and it’s present in Figma as well.

Reply:
Oh, wait – Badge vs Tip.

I’m starting to wonder whether we need Tip at all. But let’s finish Badge first.

Reply:
I think you do need Tip if you look at it's CSS but perhaps if we move some of that here and add some padding.